networks/monza/faucet/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "monza-faucet-service"
+description = "Monza Faucet Service"
```

Comment:
This goes beyond switching to well-known files, but OK.

Reply:
Well... it kind of came as a necessity because there wasn't an easy way to make the old way of running the faucet respect the ENV variables.